src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,27 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
+			$args['meta_query'] = array(
+				'relation' => 'OR',
+				array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 62030cc2830a7c413f87eecf345b8237</sub>